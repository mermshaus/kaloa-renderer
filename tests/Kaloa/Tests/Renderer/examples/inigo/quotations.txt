Other text.

[quote]This is a test[/quote]

Other text.

[quote]This is another test.

With two paragraphs.[/quote]

Other text.

[quote="Foo & Bar"]This is.

A Test.[/quote]

Other text.
