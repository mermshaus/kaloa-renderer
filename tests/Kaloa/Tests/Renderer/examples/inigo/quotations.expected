<p>Other text.</p>
<blockquote>
<p>This is a test</p>
</blockquote>

<p>Other text.</p>
<blockquote>
<p>This is another test.</p>

<p>With two paragraphs.</p>
</blockquote>

<p>Other text.</p>
<p>Foo &amp; Bar:</p>

<blockquote>
<p>This is.</p>

<p>A Test.</p>
</blockquote>

<p>Other text.</p>
